lib/src/executable/compile_stylesheet.dart

```diff
@@ -139,7 +139,7 @@ String _writeSourceMap(
     ensureDir(p.dirname(sourceMapPath));
     writeFile(sourceMapPath, sourceMapText);
 
-    url = p.toUri(sourceMapPath);
+    url = p.toUri(p.relative(sourceMapPath, from: p.dirname(destination)));
   }
 
   return (options.style == OutputStyle.compressed ? '' : '\n\n') +
```

Comment:
Is there a reason why the `soureMappingURL` is not appended for `compressed` css?

I am experiencing `compressed` style giving out wrong mappings when `sourcemap` is enabled. Is this a know issue?